templates/stripes/stripe-attributes.php

```diff
@@ -0,0 +1,18 @@
+<?php if (!empty($values)) : ?>
+  <div class="<?= $this->e($type, 'wrapperClasses') ?>">
+    <div class="vssl-stripe-column">
+      <div class="vssl-stripe--attributes--fieldwrap">
+        <?php foreach ($values as $key => $value) : ?>
+          <?php if (!empty($value['html']) ? $value['html'] : '') : ?>
+          <div class="vssl-stripe--attributes--field vssl-stripe--attributes--field--<?= $key ?>">
+            <div class="vssl-stripe--attributes--label"><?= $key ?>:</div>
+            <div class="vssl-stripe--attributes--value">
+              <?= $value['html'] ?>
```

Comment:
Strip tags here (constrain to inline, or inline + `<address>`)—we don't want multiple paragraphs, for example.

Reply:
(And could combine with lines 9 & 11 if it doesn't get too much longer :) )